Gopkg.lock

```diff
@@ -63,6 +63,86 @@
   revision = "5c4e22684113ffc2a77577c178189940925f9aef"
   version = "v1.5.1"
 
+[[projects]]
```

Comment:
um - revert this?

Reply:
Not sure why the changes here happened. I reverted it anyway.